pkg/virt-controller/services/template.go

```diff
@@ -1308,13 +1311,31 @@ func checkForKeepLauncherAfterFailure(vmi *v1.VirtualMachineInstance) bool {
 	return keepLauncherAfterFailure
 }
 
+func namespaceRequiresCPULimits(virtClient kubecli.KubevirtClient, labelSelector *metav1.LabelSelector, namespaceName string) bool {
+	namespace, err := virtClient.CoreV1().Namespaces().Get(context.Background(), namespaceName, metav1.GetOptions{})
+	if err != nil {
+		return false
+	}
+	selector, err := metav1.LabelSelectorAsSelector(labelSelector)
```

Comment:
Should we log this error? Can we move it before the "expensive" call?

Reply:
We should and we can, will do!